client/man/default.conf.5

```diff
@@ -107,6 +107,9 @@ This is used in development and is generally a detected value. It means that the
 .B interactive <boolean>
 Specifies whether values should be prompted for or not. The default is True.
 .TP
+.B kinit_lifetime <time format>
+Controls the lifetime of ticket obtained by users authenticating to the WebGUI using login/password. The expected format is a time duration string (for more info, see https://web.mit.edu/kerberos/krb5-1.12/doc/basic/date_format.html#duration). When the parameter is not set in default.conf, the ticket will have a duration inherited from the default value for kerberos clients, that can be set as ticket_lifetime in krb5.conf. When the ticket lifetime has expired, the ticket is not valid anymore and the GUI will prompt to re-login with a message "Your session has expired. Please re-login."
```

Comment:
I'd rather avoid using a pointer to an external URL. You can use an example instead, as was done in session_auth_duration.